dbloader/riak_loader.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import gevent
+from gevent.pool import Pool
+import json
+import random
+import riak as r
+import time
+from . import Loader
+from . import logger
+
+
+class RiakLoader(Loader):
+    ''' Class for load testing Riak. '''
+
+    def __init__(self):
```

Comment:
The constructor for the loader should take at least protocol, server and port as parameters and assign to self.param_name because we are creating instance of the loader in dbloader.py as:
ldr = r.RiakLoader(args.protocol, args.server, args.port)
and later on we are taking the values out of self:
self.protocol, self.host.
Similar to the postgres loader's constructor: https://github.com/phil-hildebrand/dbloader/blob/de87a12036a3d9c7571d3e0263e79d3d2df55ff2/dbloader/postgresql_loader.py#L17

Reply:
👍